cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less

```diff
@@ -144,7 +144,8 @@
         Main section with output
         --------------------------- */
   .rate-checker-heading {
-    margin-top: unit(40px / @base-font-size-px, em);
+    margin-top: unit(20px / @base-font-size-px, em);
+    margin-bottom: 1em;
```

Comment:
I don't really see a reason for that. I personally despise the unit fiddling, which imo only makes sense if you, eg, get a design spec with all the spacing laid out in pixels which you then convert to ems in this way. When you're working in browser to dial in a spacing that works, I don't think it makes sense to "unwind" it back out into the unit version.

Reply:
On the other hand, sure! Done in fc0c14a

Reply:
Yeah they have slightly different behaviors. If a user bumped their browser's default font-size up to `20px`, using the unit it would be `0.8em`, or 16px in height, whereas using the straight `1em` it would be 20px in height. The unit helper makes it always 16px, no matter the base font size.

Reply:
I don't think that's true! This is all done at compile time via LESS, so `unit(16px / @base-font-size-px, em);` just gets built to `1em;` in the output css (you can confirm this in static_built/apps/owning-a-home).

Reply:
Blergh true! I guess what I should have said is _if `@base-font-size-px` was overridden_. I thought the original intent was that it would be tied into the user's browser default size, but looking into it and at https://github.com/cfpb/design-system/blob/main/packages/cfpb-core/src/base.less#L13, I don't see how that could be linked. Hmm perhaps something to revisit.

At any rate, in this case, I think it's good to at least have the margin-top and margin-bottom follow the same format, as you have done.